recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -0,0 +1,217 @@
+        <time
+          class="d-fs-100 d-lh-300 d-fc-tertiary d-fw-normal d-ml4 d-fl-shrink0"
+        >
+          {{ time }}
+        </time>
+      </div>
+      <!-- Default content slot -->
+      <span class="content-text-wrapper-class">
+        <slot />
+      </span>
+    </article>
+
+    <template #bottom>
+      <div class="d-d-flex d-fw-wrap">
+        <slot name="reactions" />
+      </div>
+      <!-- Threads -->
+      <slot name="threading" />
+    </template>
+
+    <!-- Action menu -->
+    <template #right>
+      <div
+        v-show="isActive"
+        class="d-ps-absolute d-tn16 d-r12"
+      >
+        <dt-lazy-show
+          :appear="true"
+          transition="fade"
+          :show="isActive"
+        >
+          <slot name="menu" />
+        </dt-lazy-show>
+      </div>
+    </template>
+  </dt-list-item>
+</template>
+
+<script>
+import {} from './feed_item_row_constants.js';
+import { DtAvatar } from '@/components/avatar';
+import { DtLazyShow } from '@/components/lazy_show';
+import { DtListItem } from '@/components/list_item';
+
+export default {
+  name: 'DtRecipeFeedItemRow',
+
+  components: {
+    DtAvatar,
+    DtLazyShow,
+    DtListItem,
+  },
+
+  mixins: [],
+
+  inheritAttrs: false,
+
+  props: {
+    /**
+     * Show avatar, show header text or dont show left time and vice versa
+     */
+    showHeader: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Show the avatar of the user, if this is not passed in, the initials would display
+     */
+    avatarImageUrl: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * The display name of the sender
+     */
+    displayName: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * time string displayed as is.
+     * Shown on the header when showHeader is true
+     */
+    time: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * short time string without AM/PM displayed as is.
+     * Shown on the left of feed item when showHeader is false and isActive is true
+     */
+    shortTime: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * displays a darked background on the row.
+     */
+    isActive: {
+      type: Boolean,
+      default: false,
+    },
+
+    // TODO: implement as part of DT-1157
+    // state: {
+    //   type: String,
+    //   default: 'normal',
+    // },
+  },
+
+  emits: [
+    /**
+     * Fires when hovered over feed row
+     *
+     * @event onHover
+     * @type {Boolean}
+     */
+    'onHover',
+
+    /**
+     * Fires when focused over feed row
+     *
+     * @event onFocus
+     * @type {Boolean}
+     */
+    'onFocus',
+  ],
+
+  computed: {
+    avatarInitials () {
+      const name = this.displayName.split(' ');
+      if (name.length >= 2) {
+        return name[0].charAt(0) + name[1].charAt(0);
+      } else if (name && name[0].charAt(0) != null) {
+        return name[0].charAt(0);
+      }
+
+      return '';
+    },
```

Comment:
We will be updating the avatar itself to extract initials like this. But we don't have it yet, so this is fine for now.